staging/src/k8s.io/cli-runtime/pkg/printers/json.go

```diff
@@ -48,7 +49,8 @@ func (p *JSONPrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 		if err != nil {
 			return err
 		}
-		_, err = w.Write(data)
+		stringLiteral := strconv.QuoteToASCII(string(data))
```

Comment:
is the issue really contained to watch events output by the json printer?

I'm surprised it doesn't manifest in many places (output of `kubectl describe`, `kubectl get`, etc)

Reply:
@liggitt do you suggest changing the `_, err = w.Write(data)` on line 79 instead?

Reply:
I think there are probably _lots_ of paths that let data in free-form string fields hit the console... -o json, -o yaml, maybe simple kubectl get / kubectl describe commands, etc